iDiff/cmd/root.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	goflag "flag"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var RootCmd = &cobra.Command{
+	Use:   "iDiff",
+	Short: "iDiff is an image differ tool.",
+	Long:  `iDiff is an image differ tool.`,
+	Run: func(command *cobra.Command, args []string) {
+		fmt.Println("Root command started")
```

Comment:
Eventually we'll probably want this to be wrapped in some kind of log level (so this isn't printed every time), but we can leave it for now.